CHANGELOG.md

```diff
@@ -22,7 +22,7 @@
 * Fix a minor race condition when backing up Realm files before a client reset which could have lead to overwriting an existing file. ([PR #7341](https://github.com/realm/realm-core/pull/7341)).
  
 ### Breaking changes
-* None.
+* SyncManager no longer supports reconfiguring after calling reset_for_testing(). SyncManager::configure() has been folded into the constructor, and reset_for_testing() has been renamed to tear_down_for_testing(). ([PR #7351](https://github.com/realm/realm-core/pull/7351))
```

Comment:
I wonder if this should be under internals section. Are users supposed to use reset_for_testing?

Reply:
It's a breaking change for the SDKs which use it in their tests.

Reply:
The SDKs *are* the users of core. They should not copy and paste this into their own changelogs.